packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -70,7 +72,11 @@ cp -a .%{gem_dir}/* \
 mkdir -p %{buildroot}%{gem_extdir_mri}/%{gem_name}
 
 # hack to get the C extension where we expect it to be
+%if 0%{?rhel} >= 8
```

Comment:
Let's reverse the if-else check such that it uses the SCL path if its equal to RHEL 7 and falls back to standard paths as the else condition.